freqtrade/plugins/pairlist/__init__.py

```diff
@@ -0,0 +1 @@
+from .LiquidityPairList import LiquidityPairList  # noqa: F401
```

Comment:
No other pairlist is re-exported here - neither should this one (it has adverse affects on pairlists if they're not used).